pyproject.toml

```diff
@@ -33,7 +33,6 @@ dependencies = [
   "pyarrow",
   "typing_extensions",
   "scipy",
-  "portpicker",
```

Comment:
Do we not want to pick an open port by default if the user does not specify a port and the default port is empty?

Reply:
yea, because app and exporter need to be in sync, i couldn't think of a good way to do that using random ports

Reply:
Good point.